src/VisualStudio/CSharp/Impl/Options/Formatting/CodeStylePage.cs

```diff
@@ -325,116 +270,76 @@ private static void CSharpFormattingOptions_GenerateEditorconfig(OptionSet optio
                     editorconfig.AppendLine();
                 }
             }
+
+            Dictionary<Option<bool>, string> OptionToValue()
+            {
+                return new Dictionary<Option<bool>, string>()
+                {
+                { CSharpFormattingOptions.SpaceWithinOtherParentheses, "control_flow_statements" },
+                { CSharpFormattingOptions.SpaceWithinExpressionParentheses, "expressions" },
+                { CSharpFormattingOptions.SpaceWithinCastParentheses, "type_casts" }
+                };
+            }
```

Comment:
you made a local function to return data that never changes.
1. you could just make this a local.
2. or you could just make this a static member of the containing type.